io.catenax.packing_list/1.0.0/PackingList.ttl

```diff
@@ -0,0 +1,155 @@
+    samm:characteristic :DateCharacteristic;
+    samm:exampleValue "2023-02-11"^^xsd:date.
+:listItems a samm:Property;
+    samm:preferredName "List Items"@en;
+    samm:description "Set of items part of the packing list. "@en;
+    samm:characteristic :SetOfListItems.
+:senderAddress a samm:Property;
+    samm:preferredName "Sender Address"@en;
+    samm:description "Sender of the delivery, including the name and address. "@en;
+    samm:characteristic :SenderAddressCharacteristic.
+:receiverAddress a samm:Property;
+    samm:preferredName "Receiver Address"@en;
+    samm:description "Receiver of the delivery, including the name and address. "@en;
+    samm:characteristic :ReceiverAddressCharacteristic.
+:DateCharacteristic a samm:Characteristic;
+    samm:preferredName "DateCharacteristic"@en;
+    samm:dataType xsd:date.
+:SetOfListItems a samm-c:Set;
+    samm:preferredName "Set Of List Items"@en;
+    samm:description "Set Of List Items of the packing list. "@en;
+    samm:dataType :ListData.
+:ListData a samm:Entity;
+    samm:preferredName "List Data"@en;
+    samm:description "Catena-X ID and data of the packing list item."@en;
+    samm:properties (:quantity :position [
+  samm:property :itemDescription;
+  samm:optional "true"^^xsd:boolean
+] :itemName [
+  samm:property :catenaXId;
+  samm:optional "true"^^xsd:boolean
+]).
+:quantity a samm:Property;
+    samm:preferredName "Quantity"@en;
+    samm:description "Quantity of the item. "@en;
+    samm:characteristic :QuantityCharacteristic.
+:position a samm:Property;
+    samm:preferredName "Position"@en;
+    samm:description "Numbering of all positoins or items in the packing list. "@en;
+    samm:characteristic :PositionCharacteristic;
+    samm:exampleValue "1"^^xsd:int.
+:itemDescription a samm:Property;
+    samm:preferredName "Item Description"@en;
+    samm:description "Describes the item in the packing list. "@en;
+    samm:characteristic samm-c:Text.
+:itemName a samm:Property;
+    samm:preferredName "Item Name"@en;
+    samm:description "Item Name from a packing list. Should it be a serial number (CatenaX-ID?)"@en;
+    samm:characteristic samm-c:Text;
+    samm:exampleValue "Valve".
+:catenaXId a samm:Property;
+    samm:preferredName "Catena-X ID"@en;
+    samm:description "The Catena-X ID of the given part (e.g. the assembly), valid for the Catena-X dataspace."@en;
+    samm:characteristic ext-uuid:UuidV4Trait;
+    samm:exampleValue "urn:uuid:055c1128-0375-47c8-98de-7cf802c3241d".
+:QuantityCharacteristic a samm:Characteristic;
+    samm:preferredName "Quantity Characteristic"@en;
+    samm:description "Describes the quantity of the itemby providing a quantity value and the measurement unit in which the quantity is measured."@en;
+    samm:dataType :Quantity.
+:PositionCharacteristic a samm:Characteristic;
+    samm:preferredName "Position Characteristic"@en;
+    samm:dataType xsd:int.
+:Quantity a samm:Entity;
```

Comment:
can the shared quantity aspect be used?

Reply:
No, as we can not limit the measurement object to one.